analyzer/src/test/kotlin/PackageManagerTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2017 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.util
+
+import com.here.ort.analyzer.PackageManager
+import com.here.ort.analyzer.managers.*
+
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.matchers.shouldThrow
+import io.kotlintest.specs.WordSpec
+
+import java.io.File
+
+class PackageManagerTest : WordSpec({
+    val projectDir = File("src/funTest/assets/projects/synthetic/all-managers")
+
+    "findManagedFiles" should {
+        "find all managed files" {
+            val result = PackageManager.findManagedFiles(projectDir)
+
+            result.size shouldBe PackageManager.ALL.size
```

Comment:
This check maybe is a tad bogus because a single package manager could define multiple definition files to match. Maybe add a code comment that we only test for one definition file per package manager here?

Reply:
I have added a comment and changed the test a bit.